pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} --module-name ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} --module-name setup`
+
+- Execute a command (using the command module) on a group of hosts:
+
+`ansible {{group}} --module-name command --args '{{my command}}'`
+
+- Execute a command with administrative priviledges:
+
+`ansible {{group}} --become --ask-become-pass --module-name command --args '{{my command}}'`
+
+- Execute a command using a custom inventory file:
+
+`ansible {{group}} --inventory-file {{path/to/inventory/file}} --module-name command --args '{{my command}}'`
```

Comment:
Not sure what you mean by "there", but IIRC it was not documented in any permanent place. I made a suggestion to add that recommendation to contributing.md, but the ensuing discussion led to some downsides being pointed out, so we dropped the idea of adopting such a guideline. If I find the link to that discussion I'll add it here.